resources/views/category/itemcategories.blade.php

```diff
@@ -0,0 +1,77 @@
+@extends('layouts.app')
+
+@section('title', 'Item Category')
+
+@section('styles')
+
+<style> 
+    body {
+        margin: 0;
+        padding: 0;
+        background-color: #99ccff;
+    }
+    
+    .btn-color-sort {
+        background-color: #FFFFFF;
+    }
+
+    .btn-color-close {
+        background-color: #6699CC;
+    }
+</style>
+
+@endsection
+
+@section('content')
+<div class="container-fluid">
+    <div class="row justify-content-center mt-1">
+        <div class="col-12">
+            <h1 class="h1 text-center fw-bold">ITEM CATEGORIES</h1>
+        </div>
+    </div>
+
+    <div class="row mt-3">
+        <div class="col-10 text-end">
+            <form method="GET" action="#">            {{-- Switch between ascending and descending order --}}
```

Comment:
if this is sort you can also create a route and then remove this form method get, if the button is clicked it will go to the route and then the route have function that will sort the items

Reply:
Thank you for confirming. Regarding the current sorting, it is simply a switch between ascending order (date the category was created) and descending order. In the future, if we decide to use other types of sorting as you advised, we would like to discuss this with everyone and implement it each time.

Reply:
I created a separate css folder for this task, but it is not reflected (it has been deleted). It was not reflected even if I created and compiled the css folder and SASS file on the register page of the previous task. Since it was not reflected this time, I wrote it in the file. I will try the CSS folder again in the next task.